### PR TITLE
change hubdocs URL to hubverse.io

### PR DESCRIPTION
**Pull request by zkamvar**

### CODE REVIEW

File: R/scatter_plot_utils.R [hunk 1]
```diff
@@ -299,7 +299,7 @@ static_proj_data <- function(plot_model, df_point, df_ribbon,
 #' without target data.
 #' Simple projection model output are defined as projection associated with one
 #' particular set of "tasks_ids" value. For more information, please refer to
-#' [HubDocs website](https://hubdocs.readthedocs.io/en/latest/format/tasks.html)
+#' [HubDocs website](https://hubverse.io/en/latest/format/tasks.html)
```
Comment (LucieContamin) [inline R/scatter_plot_utils.R:302]: The page has change to:

```
[HubDocs website](https://hubverse.io/en/latest/format/tasks.html)
```

File: vignettes/plot_projection.Rmd [hunk 2]
```diff
@@ -30,7 +30,7 @@ Currently, the function can plot only quantile data, with the possibility to
 add "median" information from the model projections.
 
 For more information about the Hubverse standard format, please refer to the
-[HubDocs website](https://hubdocs.readthedocs.io/en/latest/format/tasks.html).
+[HubDocs website](https://hubverse.io/en/latest/format/tasks.html).
```
Comment (LucieContamin) [inline vignettes/plot_projection.Rmd:33]: `[HubDocs website](https://hubverse.io/en/latest/user-guide/tasks.html).`

**Review by LucieContamin — COMMENTED** [review]
Some new links are not working, I suggest new link instead

**Comment by zkamvar** [comment]
I've updated the links. Thank you for catching them! It turns out they have been invalid for a while, but that's the nature of setting up documentation.

**Review by LucieContamin — APPROVED** [review]
Thanks for the update, it looks all good to me now!

**Pull request merged**